pkg/supervisor/supervisor_unix.go

```diff
@@ -36,9 +36,9 @@ const (
 
 type unixPID int
 
-// killPid signals SIGTERM to a PID and if it's still running after
-// s.TimeoutStop sends SIGKILL.
-func (s *Supervisor) killPid(pid unixPID) error {
+// killPid tries to gracefully terminate a PID. If it's still running after
+// s.TimeoutStop the process will be terminated forcibly.
+func (s *Supervisor) killPid(ph procHandle) error {
```

Comment:
I think it would make more sense to avoid the term PID here entirely because the interface is an abstraction that doesn't include the PID itself, even though it's there underneath. 

```suggestion
// killProcess tries to gracefully terminate a Process. If it's still running after
// s.TimeoutStop the process will be terminated forcibly.
func (s *Supervisor) killProcess(ph procHandle) error {
```

Reply:
Alright, done.